### PR TITLE
Reflect camelot in pypdf_table_extraction namespace

### PR DESCRIPTION
**Pull request by foarsitter**
This is the elaboration of #9 and the first step in rebranding. 

I copied the file structure to `/pypdf_table_extraction` and imported all the objects from `camelot` where needed. This way I tried to achieve to be completely backwards compatible. 

```python
from camelot.backends.poppler_backend import PopplerBackend
from pypdf_table_extraction.backends.poppler_backend import PopplerBackend

from camelot.handlers import Lattice,Stream
from pypdf_table_extraction.parsers import Lattice,Stream
```

### CODE REVIEW

**Review by bosd — APPROVED** [review]
LGTM,
Maybe put line end at test_rename.py 😉

**Comment by foarsitter** [comment]
@MartinThoma can I have your opinion here?

File: pyproject.toml
Comment (chapmanjacobd) [inline pyproject.toml:59]: probably better to do `pdf_table_extraction` or `pypdf_table_extraction`

Comment (bosd) [reply pyproject.toml:59]: Should we go with `pypdf_table_extraction`?

Comment (stefan6419846) [reply pyproject.toml:59]: AFAIK using `pypdf_table_extraction` would allow to use the same name for both calling the script directly and calling the module through `python -m pypdf_table_extraction`, which would avoid confusion.

**Comment by bosd** [comment]
Is it ok, to move forward with this one and merge?

File: pyproject.toml
Comment (bosd) [reply pyproject.toml:59]: I agree, calling the script with the same name as for the module is the way to go 


There is already another module named pdf_table_extraction so using that name would also lead to confusion.

**Comment by bosd** [comment]
On a second thought, Let's merge this, after we merged all the open PR's from the old repo's here.
(As some are some big refactors, and merging them here is already a lot of work)

File: pyproject.toml
Comment (bosd) [reply pyproject.toml]: Added a commit to make it `pypdf_table_extraction`

**Comment by bosd** [comment]
Just tested this, added some fixes.
This one has been sitting here some while.
So Let's go!!! :sparkles: :rocket:

**Pull request merged**
